src/KafkaFlow.SchemaRegistry/ConfluentDeserializerWrapper.cs

```diff
@@ -1,20 +1,20 @@
+using System;
+using System.Collections.Concurrent;
+using System.IO;
+using System.Threading.Tasks;
+using Confluent.Kafka;
+using Microsoft.IO;
+
 namespace KafkaFlow
 {
-    using System;
-    using System.Collections.Concurrent;
-    using System.IO;
-    using System.Threading.Tasks;
-    using Confluent.Kafka;
-    using Microsoft.IO;
-
     /// <summary>
     /// A wrapper to call the typed Confluent deserializers
     /// </summary>
     public abstract class ConfluentDeserializerWrapper
     {
-        private static readonly RecyclableMemoryStreamManager MemoryStreamManager = new();
+        private static readonly RecyclableMemoryStreamManager _memoryStreamManager = new();
```

Comment:
This is a conflicting one.

[C# guidelines](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names#naming-conventions) say that static fields should have the "s_" prefix
> Static fields start with s_. Note that this isn't the default Visual Studio behavior, nor part of the [Framework design guidelines](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-fields), but is [configurable in editorconfig](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/naming-rules).

[.NET Framework guidelines](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-fields) say otherwise.
> For example, do not use "g_" or "s_" to indicate static fields.

Aspire which is a current project is using the "s_" convention. We need to decide what do use:
https://github.com/search?q=repo%3Adotnet%2Faspire%20private%20static%20readonly&type=code

It is also configurable using .editorconfig:
https://github.com/dotnet/aspire/blob/main/.editorconfig#L66

Reply:
IMO, we should use "s_" since it will be future proof. does it make any sense?

Reply:
The current .NET implementations use the `s_` prefix for static variables, so I would say to use it as well.
Example:
https://github.com/dotnet/runtime/blob/main/src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs#L59

Reply:
I also agree with using "_s" for static fields

Reply:
I've followed with the change using "_s" for static fields. Please check.

Reply:
Nice. Good job